fMRIVolume/scripts/MotionCorrection.sh

```diff
@@ -43,6 +56,32 @@ case "$MotionCorrectionType" in
     ;;
 esac
 
+# Run nonlinear registration if needed
+
+# If registering across runs, perform nonlinear registration if requested.
+# (If using linear registration, don't need to do anything extra here, since
+# linear registration is handled implicitly via the motion correction).
+# Note that if registering across runs, the "$Scout" input to MotionCorrection will
+# be the *reference* scout image (by construction in GenericfMRIVolume).
+
+if [ "${fMRIReferenceReg}" == "nonlinear" ] ; then
+  verbose_echo " ... computing nonlinear transform to reference"
+  verbose_echo "     ... generating bold average"
+  # Generating a mean image to increase SNR to noise ratio when registering to scout.
```

Comment:
"increase SNR to noise ratio" is redundant.  Either "increase SNR", or "increase signal-to-noise ratio"

Reply:
done.